go-controller/pkg/config/config.go

```diff
@@ -527,6 +530,12 @@ var CommonFlags = []cli.Flag{
 		Usage:       "Adds multicast support. Valid only with --init-master option.",
 		Destination: &EnableMulticast,
 	},
+	&cli.IntFlag{
+		Name:        "metrics-interval",
+		Usage:       "The Interval at which ovnkube & ovn metrics are collected",
+		Destination: &MetricsScrapeInterval,
```

Comment:
This should be added to one of the config objects, so it can be specified via config file instead, rather than being command-line only.